File: FCUBEAPP/src/app/models/dprvehiplacedmodel.ts
import { Dprdtlsmodel } from "./dprdtlsmodel";

export class Dprvehiplacedmodel {
    vehiclePlacedId : string = "";
    dprId : string = "";
    dprDate : string = "";
    payParty: string = "";
    partyName : string = "";
    origin : string = "";		
    fromPlace : string = "";
    destination : string = "";		
    toPlace : string = "";
    vehicleEngagedBy : string = "";
    brokerId : string = "";
    brokerName:string = "";
    vehicleNo : string = "";
    vehOwnerName : string = "";
    vehAdd1 : string = "";
    vehAdd2 : string = "";
    ownerPan : string = "";
    vehOwnerMobile : string = "";
    vehInsValidDate : string = "";
    vehFitValidDate : string = "";
    vehPermitValidDate : string = "";
    driverName : string = "";
    driverMob1 : string = "";
    challanChrgWt : string = "";
    ratePerTon : string = "";
    lorryHire : string = "";
    advance1 : string = "";
    advance2 : string = "";
    advance3 : string = "";
    advanceAmt : string = "";
    balanceAmt : string = "";
    assignToStaff : string = "";
    vehicleRptDateTime : string = "";
    placementStatus : string = "";
    placementStatusRemarks: string = "";
    noofLr: string = "";
    loggedInUser : string = "";    
    createdBy: string = "";
    createdDate: string = "";
    modifiedBy: string = "";
    modifiedDate: string = "";

    dprDtls: Dprdtlsmodel[] = [];    
}

File: FCUBEAPI/Consignment/Models/DprVehiPlacedModel.cs
namespace Consignment.Models
{
    public class DprVehiPlacedModel
    {
        public string? VehiclePlacedId { get; set; }
        public string? DprId { get; set; }
        public string? DprDate { get; set; }
        public string? PayParty { get; set; }
        public string? PartyName { get; set; }
        public string? Origin { get; set; }
        public string? FromPlace { get; set; }
        public string? Destination { get; set; }
        public string? ToPlace { get; set; }    				
        public string? VehicleEngagedBy { get; set; }
        public string? BrokerId { get; set; }
        public string? BrokerName { get; set; }        
        public string? VehicleNo { get; set; }
        public string? VehOwnerName { get; set; }
        public string? VehAdd1 { get; set; }
        public string? VehAdd2 { get; set; }
        public string? OwnerPan { get; set; }
        public string? VehOwnerMobile { get; set; }
        public string? VehInsValidDate { get; set; }
        public string? VehFitValidDate { get; set; }
        public string? VehPermitValidDate { get; set; }
        public string? DriverName { get; set; }
        public string? DriverMob1 { get; set; }						
        public string? ChallanChrgWt { get; set; }
        public string? RatePerTon { get; set; }
        public string? LorryHire { get; set; }
        public string? Advance1 { get; set; }
        public string? Advance2 { get; set; }
        public string? Advance3 { get; set; }
        public string? AdvanceAmt { get; set; }
        public string? BalanceAmt { get; set; }
        public string? AssignToStaff { get; set; }
        public string? VehicleRptDateTime { get; set; }
        public string? PlacementStatus { get; set; }
        public string? PlacementStatusRemarks { get; set; }
        public string? NoofLr { get; set; }        
        public string? LoggedInUser { get; set; }
        public string? CreatedBy { get; set; }
        public string? CreatedDate { get; set; }
        public string? ModifiedBy { get; set; }
        public string? ModifiedDate { get; set; }
        public List<DprDtlModel> DprDtls { get; set; }

    }
}
